bcpandas/utils.py

```diff
@@ -70,6 +70,12 @@ def bcp(
         auth = ["-T"]
     else:
         auth = ["-U", quote_this(creds.username), "-P", quote_this(creds.password)]
+    if creds.odbc_kwargs:
+        kwargs = {k.lower(): v for k, v in creds.odbc_kwargs.items()}
+        false_values = ("n", "no", "f", "false", "off", "0")
+
+        if "encrypt" in kwargs:
+            auth += [f"-Y{'m' if kwargs['encrypt'] not in false_values else 'o'}"]
```

Comment:
Afaik, this setting is not available on Windows so we should set this conditionally.

Reply:
It actually might make sense to throw an error in this case. Ideally it behaves just like sqlalchemy would.

Reply:
I'll check what SQLAlchemy is doing, I'll get my hands on a Windows instance later today. I think it simply ignores the query parameter though.